src/woo_publications/api/urls.py

```diff
@@ -0,0 +1,34 @@
+from django.urls import include, path, re_path
+from django.views.generic import RedirectView
+
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularJSONAPIView,
+    SpectacularRedocView,
+)
+from rest_framework import routers
+
+router = routers.DefaultRouter()
+
+
+urlpatterns = [
+    path("docs/", RedirectView.as_view(pattern_name="api-docs")),
```

Comment:
please specify the `app_label = "api"` so that the API routes are properly namespaced with `api:`

```suggestion
router = routers.DefaultRouter()

app_label = "api"

urlpatterns = [
    path("docs/", RedirectView.as_view(pattern_name="api:api-docs")),
```